packages/angular-material/example/app/app.component.ts

```diff
@@ -71,14 +71,14 @@ const itemTester: UISchemaTester = (_schema, schemaPath, _path) => {
       </button>
     </div>
     <jsonforms
-      [data]="selectedExample.data"
+      [(data)]="selectedExample.data"
```

Comment:
Do we want two-way binding here?

Reply:
yes, to showcase that the data is updated, we show this value in a div above the form